shared/components/empty-content/README.md

```diff
@@ -44,6 +43,7 @@ render: function() {
 * <strong>actionURL</strong> — (optional) `href` value for the primary action button.
 * <strong>actionCallback</strong> — (optional) `onClick` value for the primary action button.
 * <strong>actionTarget</strong> - (optional) If ommitted, no target attribute is specified.
+* <strong>featureExample</strong> - (optional) If you initialize the component with this property, containing jsx, it will be added in the bottom of the Empty Content component as an example of what the user could be viewing if there were any content.
```

Comment:
hmm, using props.children would be .. more tidy, yeah. The problem is that in some places we are abstracting the instantiation of <emptyComponent> (well, not the instantiation per se, but the properties it receive), so it would be a little messy to get the feature example separately. 

For example, in `plugins/main` we are using a helper to identify any access error that could appear, and the output of that helper is feed directly to a <EmtpyContent>. That way, using  `featureExample` as a prop, we can define that feature example directly in the object returned by the helper... the other way we would have to have two different methods, one for the props and other for the featureExample. So even if the structure of EmptyContent would be nicer using its children, I personally prefer to keep featureExample as a prop, just because its use would be easier in some cases :)